src/routes/index.js

```diff
@@ -29,6 +29,10 @@ router.use((req, res, next) => {
   }
 });
 
+router.get('/', (req, res) => {
```

Comment:
This is just a temp `get` request for information to display on the deployed page